contracts/contracts/svg/partials/background.hbs

```diff
@@ -0,0 +1,20 @@
+<g style="filter: hue-rotate({{uint16 hue}}deg);">
```

Comment:
Safari for some reason does not apply this `hue-rotate`, looked into it for a few minutes and didn't see any quick solutions. Seems like Safari handles svg color/filters pretty differently than Chrome 😑